### DISCUSSION TITLE
Proposal: design change to the dashboard job cards

### DISCUSSION THREAD

**Discussion by duckbytes** [platelet-app/platelet#145] [Ideas]
One of the most common pieces of feedback from the trial is that users would like to see more information on the dashboard cards.

The design of the current cards used in the task history view might be a good starting point for an improved design.

![image](https://user-images.githubusercontent.com/32309223/233205652-80f13821-f747-4715-84b1-26aa046219ce.png)

This card is more compact and uses context clues to show information rather than over labelling. A narrower design could be made for the dashboard that makes more use of vertical space.

I would most likely remove the status chip (this is inferred from the column it's in) and add colour coding to the card as well. I'm also unsure whether to display full names of assignees in the card or just show the circle avatars like in the current design.

Current design for reference:
![image](https://user-images.githubusercontent.com/32309223/233206075-a7cbaebb-9f3c-4393-adbf-cfa5e9b443c6.png)

Finally the task history card shows the time the job is created rather than the time of call. Cards are sorted and divided by the time created field rather than time of call that is displayed on the card. This can be a bit confusing when the time of call doesn't match the time created. To keep things consistent it might be best to just display the time created on the card and let the time of call be visible only when the card is opened.

**Comment by Joetooker95** [comment]
This looks like a good proposed design and would show everything I think we would want to see without having to go into the task. Good work Theo 👍🏻

**Reply by duckbytes** [reply]
hah thanks Joe for your speedy feedback!

**Comment by duckbytes** [comment]
Pull request here with a demo build https://github.com/platelet-app/platelet/pull/148
